perfkitbenchmarker/linux_benchmarks/fio_benchmark.py

```diff
@@ -460,6 +489,9 @@ def Run(benchmark_spec):
   stdout, stderr = vm.RobustRemoteCommand(fio_command, should_log=True)
   samples = fio.ParseResults(job_file_string, json.loads(stdout))
 
+  if collect_logs:
+    vm.PullFile(vm_util.GetTempDir(), '%s_*.log' % PKB_FIO_LOG_FILE_NAME)
```

Comment:
While going over your suggestion, I remembered now why I didn't use `vm_util.VM_TMP_DIR`. I want to store the collected logs under `/tmp/pkb/runs/<run_uri>/` so they are grouped with the results for that run, `vm_util.VM_TMP_DIR` is the path to just `/tmp/pkb/` directory.